control/init.sh

```diff
@@ -1,12 +1,18 @@
-#! /bin/bash
+# This script must be run from BASH with "source control/init.sh".
```

Comment:
A sourced script douesn't need a shebang because it runs in the shell you're in.